tools/block-generator/runner/run.go

```diff
@@ -90,6 +91,9 @@ func Run(args Args) error {
 
 	defer fmt.Println("Done running tests!")
 	return filepath.Walk(args.Path, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
```

Comment:
more graceful error when the scenario file doesn't exist, as opposed to currently naked panic